src/mongoConnector.js

```diff
@@ -8,6 +8,8 @@ module.exports.connect = mongoUri => {
   const mongoDB = mongoose.connect(mongoUri, {
     useMongoClient: true,
     promiseLibrary: global.Promise,
+    user: 'chainboarduser',
+    pass: 'password',
```

Comment:
Perhaps we could store the username and password in the config-file? Generally it's not that great to have usernames and passwords floating around with source control, but I don't really see any other way. What do you think, @simsieg @tom-lichtenstein?